examples/010_harmonic_example.py

```diff
@@ -131,6 +131,8 @@
     failure_evaluator.inputs.strains_container(strain_at_phase)
     failure_evaluator.inputs.mesh(composite_model.get_mesh())
 
+    # Note: the min/max layer indices are 1-based since
+    # WB 2024 R1 (server version 7 and above)
```

Comment:
```suggestion
    # Note: the min/max layer indices are 1-based because
    # Workbench 2024 R1 (server version 7 and above) is
    # being used.
```
I'm not quite sure if what I'm suggesting is technically correct. However, I think the note is lacking in clarity. Would you please consider revising all occurrences. Thanks!